opendebates/templates/base.html

```diff
@@ -142,6 +142,9 @@
   ODebates.paths.login = "{% url 'auth_login' %}";
   ODebates.paths.register = "{% url 'registration_register' %}";
 
+  ODebates.strings = ODebates.strings || {};
+  ODebates.strings.afterQuestionSubmittedText = {{ POPUP_AFTER_SUBMISSION_TEXT|safe }};
```

Comment:
Seems okay to me; perhaps just add a comment in the code above the `json.dumps` line stating what it's doing (providing a string wrapped double quotes that's properly escaped) to avoid anyone drawing the conclusion that this is actually json.